### PR TITLE
Update individual donor donate link

### LINKED ISSUE #237: Website 'payment' flow
Access this file: https://opensourcelk.slack.com/files/UDF4HA4TZ/F03QPBA2MB6/donate_flow_change_22-jul-2022.pptx  for details on the flow:

Step 1: Go from an aid package on the home page to the detailed view of an aid package in the goal pending tab
Screen Shot 2022-07-23 at 09 38 56

Step 2: Click Donate button in the detailed aid package view 
https://donor-portal-medicinesfor-lk.vercel.app/package/9
Screen Shot 2022-07-23 at 09 40 34

Step 3: Get directed to the donors page (page is now called how to donate):
https://donor-portal-medicinesfor-lk.vercel.app/donors

How to donate

Step 4: Click on Individual Donors 

Step 5: View a page which allows individual donors to pay (page not created, content not provided yet). Please create page as a minimum, call it Medicines for Sri Lanka for now

### PR DESCRIPTION
**Pull request by dinuka-rp**
## Purpose
The purpose of this PR is to fix #237

## Goals
Link the individual donor donate button-click to open https://give.asia/charity/slredcross

### Screenshots
https://user-images.githubusercontent.com/47103883/181255270-84e989f6-a30c-49e2-959e-79b4212123b9.mov

### CODE REVIEW

File: donor-portal/src/pages/layout/info-box/index.tsx [hunk 1]
```diff
@@ -20,11 +20,16 @@ export default function InfoBox({
...
-          <button type="button">{buttonText}</button>
-        </Link>
-      )}
+      {link !== "#" &&
+        (!link.startsWith("https://") ? (
+          <Link to={link}>
+            <button type="button">{buttonText}</button>
+          </Link>
+        ) : (
+          <a href={link} target="_blank" rel="noreferrer">
```
Comment (nipuna777) [inline donor-portal/src/pages/layout/info-box/index.tsx:29]: Do we need to open this in a new tab?

**Review by nipuna777 — APPROVED** [review]
Looks good to me! 👍🏽

File: donor-portal/src/pages/layout/info-box/index.tsx (hunk 1 above)
Comment (jayasanka-sack) [reply donor-portal/src/pages/layout/info-box/index.tsx:29]: I think it's good to open external links in a new tab.

**Comment by jayasanka-sack** [comment]
@dinuka-rp Could you please resolve conflicts?

**Pull request merged**